lib/executable-config/lookup/src/Obelisk/Configs.hs

```diff
@@ -50,6 +51,9 @@ class Monad m => HasConfigs m where
 instance Monad m => HasConfigs (ConfigsT m) where
   getConfigs = ConfigsT ask
 
+getTextConfig :: HasConfigs m => Text -> m (Maybe Text)
+getTextConfig k = fmap T.decodeUtf8 <$> getConfig k
```

Comment:
Should this be `decodeUtf8With lenientDecode`?

Reply:
You probably want the backend to fail in the case that its config can't be read, though perhaps we should reflect the encoding failure somehow, I don't know.

Reply:
In that case we should probably use `fromMaybe (error $ "Failed to decode " <> T.unpack k) . decodeUtf8'`

Reply:
@3noch agreed, except I think it better to fail in the monad.